.github/workflows/workflow.yml

```diff
@@ -51,15 +108,41 @@ jobs:
         pip install tox tox-gh-actions
         # Optional packages
         pip install pyfftw  # `test_fft` runs for pyfftw when installed
+    - name: Restore Cache
+      uses: actions/cache@v4
+      with:
+        key: cached-data-${{ hashFiles('**/registry.py', '**/*workflow.yml') }}
+        restore-keys: |
+          cached-data-
+        path: .github_cache/ASPIRE-data
+        enableCrossOsArchive: true
+    - name: Verify Restored Cache Directory
+      run: ls -lhR .github_cache/ASPIRE-data
```

Comment:
Not sure which place had the test before, but if I was going to pick one, it would be here.

Reply:
Added here.